flink-table-store-spark/src/main/java/org/apache/flink/table/store/spark/SparkTable.java

```diff
@@ -33,12 +32,12 @@
 import java.util.HashSet;
 import java.util.Set;
 
-/** A spark {@link Table} for table store. */
+/** A spark {@code Table} for table store. */
```

Comment:
Why change it?

Reply:
I will change it to `org.apache.spark.sql.connector.catalog.Table`